Source/WebKit/UIProcess/Cocoa/WebProcessProxyCocoa.mm

```diff
@@ -268,5 +272,41 @@
 }
 #endif // ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
 
+#if ENABLE(REMOTE_INSPECTOR)
+void WebProcessProxy::createServiceWorkerDebuggable(WebCore::ServiceWorkerIdentifier identifier, URL&& url)
+{
+    MESSAGE_CHECK_URL(url);
+    RELEASE_LOG(Inspector, "WebProcessProxy::createServiceWorkerDebuggable");
+    if (!shouldEnableRemoteInspector())
+        return;
+    if (RefPtr serviceWorkerDebuggableProxy = ServiceWorkerDebuggableProxy::create(url.string(), identifier, WeakPtr { *this })) {
```

Comment:
`ServiceWorkerDebuggableProxy::create()` seems to return a Ref, why store in a RefPtr and then do a null check?

Reply:
Ah, good point, I have addressed this and the other issues in the latest patch.

Thanks for reviewing!